### DISCUSSION TITLE
wall surface issue

### DISCUSSION THREAD

**Discussion by lukas1760** [Jyers/Marlin#624] [Q&A] [answered]
Hello, I don't know if it's issue regarding only to my printer so I'm asking here. I had a wall surface issue , when I used compiled and flashed firmware release 1.3.1. As you can see on pictures here are stiches on surface of model. When I print the same model with my older firmware everything is perfect. Any ideas where can be a problem? Model is box and this stiches are also on rear wall. It looks like small movement on Y axis for very short time period. Many thanks.
![Inkedwall_LI](https://user-images.githubusercontent.com/50556871/116211474-21b42180-a744-11eb-871d-85f0316eb2d5.jpg)

**Comment by hkasap83** [comment]
> Hello, I don't know if it's issue regarding only to my printer so I'm asking here. I had a wall surface issue , when I used compiled and flashed firmware release 1.3.1. As you can see on pictures here are stiches on surface of model. When I print the same model with my older firmware everything is perfect. Any ideas where can be a problem? Model is box and this stiches are also on rear wall. It looks like small movement on Y axis for very short time period. Many thanks.
> 
> ![Inkedwall_LI](https://user-images.githubusercontent.com/50556871/116211474-21b42180-a744-11eb-871d-85f0316eb2d5.jpg)
> 
> 

im still using 1.3.0 release 15x15 ubl with bltouch and have this problem too. i had no problems before that release, but im also not sure if this is related with the firmware.

**Comment by Jyers** [comment, marked as answer]
Could be a pid or steps/mm issue of some kind. Nothing else about the firmware should be different, definitely update this if you find a solution as it may be able to help others in the future

**Reply by lukas1760** [reply]
steps/mm is the same. I will try to build new one more time.
